scripts_bash/1_data_creation/organize_from_tractoflow.sh

```diff
@@ -0,0 +1,84 @@
+# checked!
+
+################################################################################
+# This script is used to precess the original HCP data and organize it into the
```

Comment:
The docstring should be re-written, this is not about HCP data anymore, and there are no args provided

Reply:
Oups. Did not re-read this. Done.